Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue5695.cs

```diff
@@ -0,0 +1,96 @@
+using System;
+using System.Collections.Generic;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+#if UITEST
+using NUnit.Framework;
+using Xamarin.UITest;
+using Xamarin.Forms.Core.UITests;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 5555, "Memory leak when SwitchCell or EntryCell", PlatformAffected.iOS)]
```

Comment:
copy/paste detected =)
I'll think about something more universal for any controls when checking for memory leaks.

Reply:
I agree, I was gonna say we had a test for most of the renderers where we pushed a page with a control, and make sure it was collected, I wonder if we can do that in the dynamic gallery test you created Pavel

Reply:
@paymicro @rmarinho fyi here it is not the control causing the leak but the RemovePage method.